pkg/cdn/getter.go

```diff
@@ -0,0 +1,12 @@
+package cdn
+
+// Getter gets the details about a given baseURL/module at vsn and returns
+// the base URL of the module metadata & content:
+//
+// - {baseURL}/{module}/@v/list
```

Comment:
thinking about "{baseURL}/{module}/@v/" having list as default behavior without explicitly specifying

Reply:
can't - that's not the spec from Russ's document

Reply:
need to go through the rest of them finally. thanks